client/src/test/java/io/moquette/spi/impl/security/AESTest.java

```diff
@@ -0,0 +1,143 @@
+package io.moquette.spi.impl.security;
+
+import com.diffblue.deeptestutils.mock.DTUMemberMatcher;
+import org.junit.Test;
+import org.junit.Assert;
+import org.junit.runner.RunWith;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import java.lang.reflect.Method;
+import javax.crypto.Cipher;
+
+import static org.mockito.AdditionalMatchers.or;
+import static org.mockito.Matchers.isA;
+import static org.mockito.Matchers.isNull;
+
+@RunWith(PowerMockRunner.class)
+@PowerMockIgnore({"javax.management.*"})
+public class AESTest {
+
+    @PrepareForTest({Cipher.class, AES.class, System.class})
+    @Test
+    public void testAESEncrypt() throws Exception {
+        PowerMockito.mockStatic(System.class);
```

Comment:
Is mocking the `System.class` necessary for this method?

Reply:
Removed